.github/workflows/deploy-docs-guidelines.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Deploy Docs Guidelines
+on:
+  # Runs on pushes targeting the `docs-guildelines` branch
+  push:
+    branches: ["feature/docs-guidelines"]
```

Comment:
should we also trigger this build whenever there is a merge into main? Or will that come in a later PR against main?

Reply:
Yes we can update that when we merge `docs-guidelines` branch to main.